frontend/lib/parsing/chpl.ypp

```diff
@@ -2822,6 +2828,13 @@ var_decl_stmt:
     $$ = context->buildVarOrMultiDeclStmt(@$, $2);
     context->resetDeclState();
   }
+| var_decl_type error TSEMI
+  {
+    auto err = ErroneousExpression::build(BUILDER, LOC(@2));
+    CommentsAndStmt cs = { .comments=nullptr, .stmt=err.release() };
```

Comment:
Ditto if we could add a method about building an erroneous expression from a statement and just add that. It's nice to have as many one-call lines in the parser as possible.